android/upload_symbols.gradle

```diff
@@ -0,0 +1,256 @@
+import java.util.zip.ZipOutputStream
+import java.util.zip.ZipEntry
+
+// Custom Exception for undefined variable
+public class VariableNotFoundException extends Exception {
+    public VariableNotFoundException(String errorMessage) {
+        super(errorMessage);
+    }
+}
+// Constants
+final INSTABUG_APP_TOKEN = 'INSTABUG_APP_TOKEN'
+final INSTABUG_API_KEY = 'INSTABUG_API_KEY'
+final UPLOAD_URL = 'https://api.instabug.com/api/web/public/flutter-symbol-files/android'
+final INSTABUG_AUTO_UPLOAD_ENABLE = 'INSTABUG_AUTO_UPLOAD_ENABLE'
+ext.boundary = 'abcd' + Long.toString(System.currentTimeMillis()) * 2 + 'dcba'
+ext.lineEnd = '\r\n'
+
+/**
+ * task to upload symbols to Instabug endpoint
+ */
+task upload_symbols_task {
+    doLast {
+        try {
+            // If obfuscate flag not provided, exit the task
+            if (project['dart-obfuscation'] != 'true') {
+                println "Instabug: skipping debug symbols upload"
```

Comment:
I would suggest we don't log anything related to debug symbols, in case the user didn't run the build command with the `--obfuscate` flag. What do you think?